src/figuresClasses.ts

```diff
@@ -1,19 +1,79 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Wrong sides');
+    } else if ((a + b <= c) || (a + c <= b) || (b + c <= a)) {
```

Comment:
Use more descriptive text for errors.
Also, there is no need to use else...if as code will not go further after throwing an error anyway
![image](https://github.com/mate-academy/ts_figures_classes/assets/96409563/9e34268f-5d0a-4d1d-b172-e7b7a10702f3)